docs/GETTING-STARTED.md

```diff
@@ -0,0 +1,318 @@
+sudo dnf -y groupinstall 'C Development Tools and Libraries'
+sudo dnf install -y yum-utils
+sudo dnf install -y git
+sudo dnf install -y ShellCheck
+```
+Set selinux to Permissive.
+```
+sudo setenforce 0
+```
+#### Install docker
+
+Install version from Fedora repo or install from Docker CE upstream.
+
+Install current version from Fedora repo.
+
+```
+sudo dnf install -y docker
+```
+The current scripts and dockerfiles don't work to well with the older version of docker in the Fedora 28 repo.
+
+As an alternative, install from Docker CE upstream. First remove any old versions of docker.
+````
+sudo dnf remove docker \
+                  docker-client \
+                  docker-client-latest \
+                  docker-common \
+                  docker-latest \
+                  docker-latest-logrotate \
+                  docker-logrotate \
+                  docker-selinux \
+                  docker-engine-selinux \
+                  docker-engine
+````
+Install Docker CE
+````
+sudo dnf config-manager \
+    --add-repo \
+    https://download.docker.com/linux/fedora/docker-ce.repo
+sudo dnf config-manager --set-enabled docker-ce-edge
+sudo dnf install -y docker-ce
+````
+
+Now, add yourself to a docker group so you can run docker commands without sudo.
+```
+sudo groupadd docker
+sudo usermod -a -G docker $(whoami)
+```
+Logout and back in again. Also restart docker.
+```
+sudo systemctl restart docker
+```
+Test docker
+```
+docker run hello-world
+```
+
+#### Install kubernetes as documented in kubernetes.io
+
+The following procedure installs kubernetes as a single-node cluster
+[https://kubernetes.io/docs/setup/pick-right-solution/#local-machine-solutions]
+
+
+First install virt and kubectl. Then install golangand minikube.
+
+### Install virt and kvm
+
+```
+sudo dnf install -y qemu-kvm qemu-img libvirt libvirt-python libvirt-client virt-install virt-viewer bridge-utils
+sudo systemctl start libvirtd
+sudo systemctl enable libvirtd
+```
+
+[Documented in more detail here:](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+#### Install kubectl
+
+Install upstream kubernetes.io repo for the most recent packages.
+
+```
+sudo vi /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+```
+
+```
+sudo dnf config-manager --set-enabled kubernetes
+sudo dnf install -y kubectl
+```
+
+### Install golang
+
+[Using these instructions from the go project.](https://go-repo.io/)
+However, we will use the version of golang in the Fedora distro.
+
+```
+sudo dnf install -y golang
+    
+mkdir -p ~/go/{bin,pkg,src}
+echo 'export GOPATH="$HOME/go"' >> ~/.bashrc
+echo 'export PATH="$PATH:${GOPATH//://bin:}/bin"' >> ~/.bashrc
+source $HOME/.bashrc
+```
+To test go installation:
```

Comment:
For consistency, it may make sense to have a blank line in between the block quote sections, as you've done above.